diff --git a/node_modules/@shopify/flash-list/dist/MasonryFlashList.d.ts b/node_modules/@shopify/flash-list/dist/MasonryFlashList.d.ts
index b328c8c..e67ce4b 100644
--- a/node_modules/@shopify/flash-list/dist/MasonryFlashList.d.ts
+++ b/node_modules/@shopify/flash-list/dist/MasonryFlashList.d.ts
@@ -40,6 +40,7 @@ export interface MasonryFlashListRef<T> {
     scrollToOffset: FlashList<T>["scrollToOffset"];
     scrollToEnd: FlashList<T>["scrollToEnd"];
     getScrollableNode: FlashList<T>["getScrollableNode"];
+    recyclerlistview_unsafe: FlashList<T>["recyclerlistview_unsafe"];
 }
 /**
  * FlashList variant that enables rendering of masonry layouts.
diff --git a/node_modules/@shopify/flash-list/src/MasonryFlashList.tsx b/node_modules/@shopify/flash-list/src/MasonryFlashList.tsx
index 957cd5b..3447834 100644
--- a/node_modules/@shopify/flash-list/src/MasonryFlashList.tsx
+++ b/node_modules/@shopify/flash-list/src/MasonryFlashList.tsx
@@ -80,6 +80,7 @@ export interface MasonryFlashListRef<T> {
   scrollToOffset: FlashList<T>["scrollToOffset"];
   scrollToEnd: FlashList<T>["scrollToEnd"];
   getScrollableNode: FlashList<T>["getScrollableNode"];
+  recyclerlistview_unsafe: FlashList<T>["recyclerlistview_unsafe"];
 }
 
 /**
